src/services/PRService.ts

```diff
@@ -0,0 +1,199 @@
+import { CodeReviewAgentApp } from '../../CodeReviewAgentApp';
+import { AppSettingsEnum } from '../config/settings';
+import { GitHubAPIService, GitHubPullRequest } from './GitHubAPIService';
+import { PRPersistence, StoredPR } from '../persistence/PRPersistence';
+import { IPersistence, IPersistenceRead } from '@rocket.chat/apps-engine/definition/accessors';
+
+export class PRService {
+    private githubService: GitHubAPIService;
+
+    constructor(private app: CodeReviewAgentApp) {
+        this.githubService = new GitHubAPIService(app);
+    }
+
+    /**
+     * Fetches new PRs from all configured repositories and stores them
+     */
+    public async fetchAndStoreNewPRs(persistence: IPersistence, persistenceRead: IPersistenceRead): Promise<StoredPR[]> {
+        try {
+            const logger = this.app.getLogger();
+            
+            const owner = await this.app.getAccessors().environmentReader
+                .getSettings().getValueById(AppSettingsEnum.OWNER_NAME_ID);
+            const repositoriesRaw = await this.app.getAccessors().environmentReader
+                .getSettings().getValueById(AppSettingsEnum.REPOSITORIES_LIST_ID);
+            
+            if (!owner || !repositoriesRaw) {
+                logger.error('Owner name or repositories list not configured');
+                return [];
+            }
+            
+            const repositories = repositoriesRaw.split('\n')
+                .map(repo => repo.trim())
+                .filter(repo => repo.length > 0);
+            
+            logger.info(`Fetching PRs for ${repositories.length} repositories`);
+            
+            const newPRs: StoredPR[] = [];
+            const since = new Date(Date.now() - 12 * 60 * 60 * 1000); // Last 12 hours
```

Comment:
let's use since value from storage (like save the start time of every pipeline into db as epochms time)